File: 프로그래머스/unrated/181836. 그림 확대/그림 확대.cs
using System;
using System.Text;

public class Solution {
    public string[] solution(string[] picture, int k) 
    {
        var answer = new string[picture.Length * k];
        
        var sb = new StringBuilder();
        for(int i = 0; i < picture.Length; ++i)
        {
            foreach(char c in picture[i])
            {
                for(int r = 0; r < k; ++r)
                    sb.Append(c);
            }
            
            for(int r = 0; r < k; ++r)
                answer[k * i + r] = sb.ToString();
            
            sb.Clear();
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) {
    vector<string> answer;
    
    for (const string& picStr : picture)
    {
        string str;
        
        for (const char& c : picStr)
        {
            for (int x = 0; x < k; ++x)
                str += c;            
        }
        
        for (int y = 0; y < k; ++y)
            answer.push_back(str);
    }

    return answer;
}